src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-user-details/rgw-user-details.component.html

```diff
@@ -151,5 +151,58 @@
         </tbody>
       </table>
     </div>
+    <!-- User Rate Limit -->
+    <div *ngIf="user.user_ratelimit">
+      <legend i18n>User Rate Limit</legend>
+      <table class="cds--data-table--sort cds--data-table--no-border cds--data-table cds--data-table--md">
+        <tbody>
+          <tr>
+            <td i18n
+                class="bold w-25">Enabled</td>
+            <td class="w-75">{{ user.user_ratelimit.enabled | booleanText }}</td>
+          </tr>
+          <tr>
+            <td i18n
+                class="bold">Maximum Read Ops</td>
```

Comment:
```suggestion
                class="bold">Maximum read ops</td>
```
As observed, applicable at all other places where we have capitalized

Reply:
Thanks for the suggestion. Fixed.